code/modules/mob/living/silicon/robot/dogborg/dog_modules_vr.dm

```diff
@@ -158,6 +158,13 @@
 					reagent_volumes[T] = min(reagent_volumes[T] + 1, volume)
 	return 1
 
+/obj/item/weapon/reagent_containers/borghypo/hound/lost
+	name = "Hound hypospray"
+	desc = "An advanced chemical synthesizer and injection system utilizing carrier's reserves."
+	charge_cost = 5
+	reagent_ids = list("tricordrazine", "inaprovaline", "bicaridine", "dexalin", "anti_toxin", "tramadol", "spaceacillin", "hyperzine")
```

Comment:
This is a bit too strong. It already has far less charge cost than most borgs, and a better array of chems than most borgs, including Polaris' own lost borg analogue. I would increase charge cost to 10 (like medihounds) and give it identical chems to the Polaris lost borg

Reply:
The reason I buffed variety was mostly because loss of R&D tool, so inability to help research is made up for with better medical prowess. I'll up charge cost back, was a bit overkill on that. But chem additions were only inaprovaline (which is pretty inconsequential tbh? with bicaridine already being there, I added it mostly for flavor) and hyperzine, which is mild but acceptable with spendings on water and energy present?

Reply:
I would trim hyperzine off, at the very least. My main concern is that someone is going to probably SCA one of these things at some point.

Reply:
Alright. I'm also removing SleeveMate, since the borg is primarily expedition encounter, and having sleeve stuff over there is rather redundant/wrong-thing-encouraging. Especially with flavor being that they aren't necessarily Virgo's borg.